pkg/storaged/lvol-tabs.jsx

```diff
@@ -230,19 +246,89 @@
         return;
     }
 
+    const pvs_as_spaces = client.vgroups_pvols[vgroup.path].filter(pvol => pvol.FreeSize > 0).map(pvol => {
+        const block = client.blocks[pvol.path];
+        return { type: 'block', block, size: pvol.FreeSize, desc: "", pvol };
+    });
+
+    let initial_pvs;
+
+    if (!has_subvols)
+        initial_pvs = pvs_as_spaces;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17226-20230531-081900-df2e1047-fedora-testing-daily//Coverage/lcov/github-pr.diff.gcov.html)